lib/views/user/_signup.html.erb

```diff
@@ -28,6 +28,20 @@
     </div>
 
     <p>
+      <label class="form_label" for="user_signup_postcode">
+      <%= _('Your postcode:') %></label>
+      <%= text_field 'user_signup',
+                     'postcode', { :size => 20, :tabindex => 21 } %>
+    </p>
+
+    <p>
+      <label class="form_label" for="user_signup_province">
+        <%= _('Your province:') %></label>
+        <%= select('user_signup', 'province', province_name_options,
+                   { :tabindex => 22 }) %>
```

Comment:
Ah, these weren't right  - I've set `tabindex` in options instead of html_options